projects/plugins/jetpack/_inc/client/components/search/close-btn.jsx

```diff
@@ -0,0 +1,19 @@
+import Gridicon from 'components/gridicon';
+
+const SearchCloseButton = ( { instanceId, closeSearch, closeListener } ) => {
```

Comment:
Copied from `index.jsx`. No changes were made apart from passing the listeners and state values as props.